configuration/config-txt/video.md

```diff
@@ -156,13 +123,10 @@ The `hdmi_group` command defines the HDMI output group to be either CEA (Consume
 | 1 | CEA |
 | 2 | DMT |
 
+<a name="hdmimode"></a>
 ### hdmi_mode
 
-Together with `hdmi_group`, `hdmi_mode` defines the HDMI output format. Format mode numbers are derived from the CTA specification found [here](https://web.archive.org/web/20171201033424/https://standards.cta.tech/kwspub/published_docs/CTA-861-G_FINAL_revised_2017.pdf) 
-
-To set a custom display mode not listed here, see [this thread](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=24679).
```

Comment:
Is it worth keeping this forum link @JamesH65 ?

Reply:
That one is redundant, since it links to a forum post describing CVT modes, and this information is already present further down the page.